nginx/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM nginx:1.26.1
+
```

Comment:
Missing `apt update && apt upgrade -y`

Reply:
same as for django, there is no point to update just for the sake of it